apps/microtvm/reference-vm/README.md

```diff
@@ -15,53 +15,85 @@
-           a rule to autoconnect this device to the VM, and also attaches it to the VM>
-        4. SSHs to the VM, `cd` to the TVM root directory, and runs `test_cmd` from
-           `test-config.json`. Nonzero status means failure.
-3. If release tests fail, fix them and restart from step 1.
-4. If release tests pass: `$ ./base-box-tool.py [--provider=<provider>] release <--release-version=<version>> <--platform-version=<version>> <platform>`. Be sure you've logged
-   in to Vagrant Cloud using the `vagrant` tool.
+# microTVM Reference Virtual Machines (RVM)
+
+This directory contains Vagrant specifications that create Reference Virtual
+Machines (RVM) for use with microTVM. These machines help microTVM users
+collaborate by providing a stable reference environment to build and test
+microTVM.
+
+For more information on how to use them, see the
+[microTVM Reference VM tutorial](../../../tutorials/micro/micro_reference_vm.py).
+
+
+## microTVM developer information
+
+Each RTOS or platform (like Zephyr, FreeRTOS, etc) that integrates with microTVM
+can check-in a Reference VM in this directory to help the community collaborate.
+You should use the tools provided here to ensure a uniform release process
+across all platforms. Typically, releases need to be created by TVM committers.
+
+Generally speaking, it's expected that any integrated platform with a regression
+test checked-in to the tvm repository should also define a reference VM. If you
+want to integrate a new platform, please raise a discussion on
+[the forum](https://discuss.tvm.ai).
+
+
+## Reference VMs organization
+
+Reference VMs are organized in this directory as follows:
+
+```
+.
++-- base-box-tool.py - Reference VM build, test, and release tool.
++-- <platform>/      - One or more dirs related to the supported platform(s).
+    +-- Vagrantfile  - Vagrantfile that end-users will invoke. Should be based
+    |                  off a base box which contains dependencies other than the
+    |                  TVM python dependencies.
+    +-- base-box/    - Top-level directory which defines the base box.
+        +-- Vagrantfile.packer-template - 'packer' template Vagrantfile which
+        |                                 will be used to build the base box.
+        +-- test-config.json            - JSON file explaining how to perform
+                                          release tests to base-box-tool.py.
+```
+
+
+## Creating releases
+
+1. **Build** the base box for a given platform:
+```bash
+$ ./base-box-tool.py [--provider=<provider>] build <platform>
+```
+
+2. **Run** release tests for each platform:
+
+   A. Connect any needed hardware to the VM host machine;
+
+   B. Run tests:
+```bash
+$ ./base-box-tool.py [--provider=<provider>] test [--microtvm-platform=<platform>] [--test-device-serial=<serial>] <platform>
+```
+   This command does the following for the specified provider:
```

Comment:
can you fix the indentation here? If you look at preview file this section is not indented as part of section 2.

Reply:
Sure! Done.